src/components/views/settings/SetIntegrationManager.tsx

```diff
@@ -93,4 +94,4 @@ export default class SetIntegrationManager extends React.Component<IProps, IStat
             </div>
         );
     }
-}
+}
```

Comment:
You seem to have removed a line by accident here, could you please revert that change?